perllib/FixMyStreet/App/Controller/Report/Update.pm

```diff
@@ -548,24 +548,17 @@ sub signup_for_alerts : Private {
     my ( $self, $c ) = @_;
 
     my $update = $c->stash->{update};
+    my $user = $update->user;
+    my $problem_id = $update->problem_id;
+
     if ( $c->stash->{add_alert} ) {
         my $options = {
-            user => $update->user,
-            alert_type => 'new_updates',
-            parameter => $update->problem_id,
+          cobrand      => $update->cobrand,
```

Comment:
Style guide, as with the code this is replacing, is not to align, because otherwise there are needless diffs if the list of keys ever changes.